python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -220,7 +221,8 @@ def __init__(
             dependencies_path.exists() or packages_path.exists()
         ) and not packages_install_path.exists()
 
-        super().__init__(
+        val = cls.__model_new__(  # type: ignore
```

Comment:
this approach resolves having to type ignore every line
could add a class to inherit too, though I haven't been able to find any clever typing approaches so its basically just another flavor of untyped and likely adds an extra function call